applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py

```diff
@@ -109,6 +112,7 @@ def AssembleTestSuites():
     return suites
 
 if __name__ == '__main__':
+    KratosMultiphysics.Logger.GetDefaultOutput().SetSeverity(KratosMultiphysics.Logger.Severity.WARNING)
```

Comment:
@jcotela is this ok?
Otherwise the tests-output is very verbose...

Reply:
Sure :+1: